pkg/build/api/types.go

```diff
@@ -177,6 +190,9 @@ const (
 	// StatusReasonExceededRetryTimeout is an error condition when the build has
 	// not completed and retrying the build times out.
 	StatusReasonExceededRetryTimeout = "ExceededRetryTimeout"
+
+	// StatusReasonSkipped indicates that the build was skipped.
+	StatusReasonSkipped = "Skipped"
```

Comment:
technically "skipped" means cancelled, but we want to leave some notion there for auditing... also the cancel-build will have to have some option to set that.